src/app/docs/sort/sort-docs.component.html

```diff
@@ -0,0 +1,20 @@
+<sky-docs-demo-page
+  moduleName="SkySortModule"
+  moduleSourceCodePath="src/app/public/modules/sort/"
+  pageTitle="Sort"
+>
+  <sky-docs-demo-page-summary>
+    The sort component allows users to select sorting criteria. The component creates a button and dropdown menu to provide sorting options.
+  </sky-docs-demo-page-summary>
+
```

Comment:
We have one existing Design guideline in place at https://developer.blackbaud.com/skyux/components/sort#ux-guidelines. Do you think it's worth setting up the Design tab to get that in place? If so, I can do it. Just let me know.

Reply:
Same as above - I'd recommend we make that a separate piece of work to keep our PRs manageable.

Reply:
Sounds good. I created https://github.com/blackbaud/skyux-lists/issues/168 to track the work for this.